### ISSUE TITLE
Embedded file names

### ISSUE THREAD

**Issue by jjack80** [volexity/threat-intel#6]
Not necessarily an issue, but does there by chance exist a version of onenoteextractor that could write the extracted files to disc using their original file names?

**Comment by tlansec** [comment]
Hi there,

**TLDR**

If you want the original filenames from OneNote files we recommend you use the following library: https://github.com/DissectMalware/pyOneNote

**Explanation**

Sorry for the delay in our response. The approach we took when writing our parser was effectively to grep the file for the GUID used to denote the presence of an attached file in a OneNote document. In order to properly extract all of the properties of the embedded object (such as filenames) required a more refined understanding of the structures involved, which isn't present in our extractor at the moment.

We could port the logic from the pyOneNote project into our parser, but:

1. This feels rude as the author of the other project has done all of the legwork to properly understand the OneNote file format
2. It's probably better that we (as a community) try to maintain just one library to deal with OneNote files.

Hopefully this makes sense. 

Cheers,
Tom
